Description: fix default permissions when creating the log directory.
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/session-migration/+bug/1735929

--- a/hw/xfree86/common/xf86Helper.c
+++ b/hw/xfree86/common/xf86Helper.c
@@ -1135,10 +1135,10 @@ static void xf86_mkdir_p(char *path)
 
     while ((sep = strchr(sep + 1, '/'))) {
         *sep = 0;
-        (void)mkdir(path, 0777);
+        (void)mkdir(path, 0700);
         *sep = '/';
     }
-    (void)mkdir(path, 0777);
+    (void)mkdir(path, 0700);
 }
 
 void
